src/containers/shared/components/Transaction/AMM/AMMBid/simple.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
```

Comment:
this should be `Simple.tsx` since it's a React component

Reply:
ditto for the rest

Reply:
done